lib/gettext-setup/merge_pot.rb

```diff
@@ -0,0 +1,9 @@
+# A task to merge pot files located within the locales directory into one
+# pot file for localization. Output is 'completed.pot' in locales directory.
+module GettextSetup
+  module MergePot
+    def self.merge_pot
+      `msgcat locales/*.pot > locales/completed.pot`
```

Comment:
It seems like a good idea here to check to see if `completed.pot` (or whatever filename makes sense for the aggregated POT file) already exists. This might avoid duplicates, since this isn't idempotent in nature.

I'm not sure whether `msgcat` removes duplicates, but either way, I'd check for the prior existence of that file.